enhancements/ingress/haproxy-thread-tuning.md

```diff
@@ -0,0 +1,216 @@
+---
+title: haproxy-thread-tuning
+authors:
+  - "@rfredette"
+reviewers:
+  - "@Miciah"
+  - "@danehans"
+  - "@frobware"
+  - "@sgreene570"
+  - "@knobunc"
+  - "@miheer"
+  - "@candita"
+approvers:
+  - "@knobunc"
+  - "@miciah"
+  - "@frobware"
+  - "@danehans"
+creation-date: 2021-03-24
+last-updated: 2021-03-26
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced|informational
+see-also: [Tunalbe Router Buffer Sizes](https://github.com/openshift/enhancements/pull/449)
```

Comment:
I think these references are supposed to be formatted like this:
```suggestion
see-also:
- "/enhancements/ingress/tunable-router-buffer-sizes.md"
```
Looks like there is little consistency among existing enhancements though.  Let's sort this out separately; no need to hold up the PR for this.